assets/app/views/directives/osc-key-values.html

```diff
@@ -1,95 +1,175 @@
 <div ng-controller="KeyValuesController" class="labels">
     <div class="form-inline labels-edit" ng-show="editable">
-      <ng-form class="edit-label" name="form" novalidate>
-        <div class='form-group' ng-class="{'has-error': form.key.$error.oscKeyValid}">
-          <input class="form-control"
-                 type="text"
-                 name="key"
-                 ng-attr-placeholder="{{keyTitle}}"
-                 ng-model="key"
-                 autocorrect="off"
-                 autocapitalize="off"
-                 spellcheck="false"
-                 osc-input-validator="key">
-        </div>
-        <div class="form-group" ng-class="{'has-error': form.value.$error.oscValueValid}">
-          <input class="form-control"
-                 type="text"
-                 name="value"
-                 ng-attr-placeholder="{{valueTitle}}"
-                 ng-model="value"
-                 autocorrect="off"
-                 autocapitalize="off"
-                 spellcheck="false"
-                 osc-input-validator="value">
+      <form class="edit-label" name="form" novalidate>
+
+        <div layout row>
+
+          <div
+            flex grow="5"
+            class='form-group'
+            ng-class="{'has-error': form.key.$error.oscKeyValid}"
+            style="margin-right: 10px;">
+            <input class="form-control"
+                   type="text"
+                   name="key"
+                   ng-attr-placeholder="{{keyTitle}}"
+                   ng-model="key"
+                   autocorrect="off"
+                   autocapitalize="off"
+                   spellcheck="false"
+                   osc-input-validator="key">
+          </div>
+
+          <div
+            flex grow="5"
+            class="form-group"
+            ng-class="{'has-error': form.value.$error.oscValueValid}"
+            style="margin-right: 10px;">
+            <input class="form-control"
+                   type="text"
+                   name="value"
+                   ng-attr-placeholder="{{valueTitle}}"
+                   ng-model="value"
+                   autocorrect="off"
+                   autocapitalize="off"
+                   spellcheck="false"
+                   osc-input-validator="value">
+          </div>
+          <div
+            flex layout
+            grow="1" shrink="1" basis="auto"
```

Comment:
Right, a new page I'm working on has it even wider, which is why I asked. It would be nice if the text input expand to fill the space with no gaps.

I wonder if an approach like this wouldn't work.

http://stackoverflow.com/questions/12858979/put-input-submit-on-same-line-as-input-box#answer-12859038

Not as elegant as the flexbox solution, but might get rid of the gap.